src/ai_migrate/storage/config.py

```diff
@@ -0,0 +1,40 @@
+from enum import Enum
+from pathlib import Path
+from typing import Optional, Dict, Union
+
+from pydantic import BaseModel, Field, field_validator
+
+
+class StorageType(str, Enum):
+    LOCAL = "local"
+    GDRIVE = "gdrive"
+    S3 = "s3"
+    GCS = "gcs"
+    AZURE = "azure"
```

Comment:
do we need all of these?

Reply:
wanted to show the extensibility here. Will comment some